Rewrite to JS Object broke backend db migrations

According to the README, I have to manually make migrations for the backend.
I've used the commands from the README:
```
# apply DB migrations
docker exec -it adsycc_backenddev1_1 /bin/bash
cd /usr/src/app/
make migrations
```
Since the config file was rewritten from json to JS Object, this should be also fixed:
```
root@20916061fa73:/usr/src/app# make migrations                                                                                             
node_modules/.bin/babel-node \                                                                                                              
        node_modules/.bin/knex migrate:latest                         
module.js:472                                                         
    throw err;                                                        
    ^                                                                 
                                                                      
Error: Cannot find module './config.json'                                                                                                   
    at Function.Module._resolveFilename (module.js:470:15) 
    at Function.Module._load (module.js:418:25)            
    at Module.require (module.js:498:17)                   
    at require (internal/module.js:20:19)                  
    at Object.<anonymous> (/usr/src/app/knexfile.js:1:16)  
    at Module._compile (module.js:571:32)                  
    at loader (/usr/src/app/node_modules/babel-register/lib/node.js:144:5)                                             
    at Object.require.extensions.(anonymous function) [as .js] (/usr/src/app/node_modules/babel-register/lib/node.js:154:7)                                                                                                                    
    at Module.load (module.js:488:32)                      
    at tryModuleLoad (module.js:447:12)                    
    at Function.Module._load (module.js:439:3)             
    at Module.require (module.js:498:17)                   
    at require (internal/module.js:20:19)                  
    at initKnex (/usr/src/app/node_modules/knex/bin/cli.js:50:16)                                                      
    at Command.<anonymous> (/usr/src/app/node_modules/knex/bin/cli.js:127:17)                                          
    at Command.listener (/usr/src/app/node_modules/commander/index.js:300:8)                                           
Makefile:17: recipe for target 'migrations' failed         
make: *** [migrations] Error 1                             
```

--- a/backend/config.example.js
+++ b/backend/config.example.js
@@ -1,4 +1,4 @@
-export default {
+module.exports = {
   application: {
     name: 'Customer Center',
     host: 'customer-center.example.com'

--- a/backend/knexfile.js
+++ b/backend/knexfile.js
@@ -1,4 +1,4 @@
-const config = require('./config.json').database
+const config = require('./config.js').database
 
 module.exports = {
   development: config,

